src/test/java/hudson/plugins/logparser/LogParserTestCase.java

```diff
@@ -50,15 +48,18 @@ protected void assertLogParserCorrect(Run<?, ?> build, Job<?, ?> project, Result
         assertThat(build.getAction(LogParserAction.class)).as(LogParserAction.class.getSimpleName() + " not found")
                 .isNotNull();
                 
-        boolean aborted = Result.ABORTED.equals(build.getResult());
+        boolean aborted = Result.ABORTED == build.getResult();
         
         LogParserResult parserResult = build.getAction(LogParserAction.class).getResult();
         assertThat(parserResult).isNotNull();
         
         if (!aborted) {
-            assertThat(parserResult.getTotalErrors()).isEqualTo(1);
-            assertThat(parserResult.getTotalWarnings()).isEqualTo(1);
-            assertThat(parserResult.getTotalInfos()).isEqualTo(1);
+            assertThat(new int[] {
+                parserResult.getTotalErrors(),
+                parserResult.getTotalWarnings(),
+                parserResult.getTotalInfos(),
+            }).as("Did not evaluate all lines correctly. The result [2,2,2] may be the project log lines creation commands showing up in log twice (eg. -x passed to shell).")
+            .isEqualTo(new int[] {1, 1, 1});
```

Comment:
Improved the checking so that in case of an error you can see all three numbers at once.